pkg/detectors/slack/slack.go

```diff
@@ -25,10 +25,10 @@ var _ detectors.Detector = Scanner{}
 var (
 	defaultClient = common.SaneHttpClient()
 	tokenPats     = map[string]*regexp.Regexp{
-		"Slack Bot Token":               regexp.MustCompile(`xoxb\-[0-9]{10,13}\-[0-9]{10,13}[a-zA-Z0-9\-]*`),
```

Comment:
I believe these are the "legacy" format. It may be best to track the pattern separately, rather than loosening the format.

e.g., https://github.com/gitleaks/gitleaks/blob/4e43d1109303568509596ef5ef576fbdc0509891/cmd/generate/config/rules/slack.go#L153

Reply:
I don't think loosening the regex should have any negative impact. It already matches on 2+[2 or more] subset of digits. I just changed it to 1+subset of digits.

Reply:
hi @rgmz ,

These are not 2 distinct patterns. The current regex already captures patterns such as xoxb-[0-9]-[0-9]-Abckdkhf and xoxb-[0-9]-[0-9]-[0-9]-Abckdkhf and xoxb-[0-9]-[0-9]-[0-9]-Abckdkhf and so on. I dont think that xoxb-[0-9]-Akjkdf is a new pattern. It's already a part of an existing pattern.  Also, normally when format changes, a new type of token pattern is introduced.so, I don't think this was the old format.

Also, Current regex captures false positives too. 

```
xoxb-12345678901-0987654321-your-token-here
xoxb-1234567890-1234567890

```

Do you want me to make this v1 and v2? I don't see the point!

Reply:
> These are not 2 distinct patterns.

They are. Slack changed their bot token patterns and no longer generates tokens like `xoxb-93316802544-VsDuvHwRiPgxppomh0nxazdE`.

> Also, Current regex captures false positives too.

It could certainly be improved. My point is that this change would capture net-new invalid patterns that wouldn't be detected if there are two separate patterns.

> Do you want me to make this v1 and v2? I don't see the point!

Why not add a new entry to `tokenPats`?

Reply:
I am with @rgmz. It will simplified the maintenance and allow tracking for different versions.